docs/user-guide/side-bar.md

```diff
@@ -101,3 +105,14 @@ In order to have information about one layer only the user can select the layer
 
 <img src="../img/side-bar/layer.gif" class="ms-docimage"/>
 
+### Floating Identify Tool 
+
+On [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) the user can activate the **Floating Identify tool** instead of the default [Identify tool](side-bar.md#identify-tool) through which a mobile panel opens to retrieve information about an object on the map.
+
+In order to activate the *Floating Identify Tool* the user can select the <img src="../img/button/setting_button.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu) <img src="../img/button/burger.jpg" class="ms-docbutton" />. Here he can select the **Hover** option through the *Trigger event for Identify* dropdown menu.
+
+<img src="../img/side-bar/setting-hover.jpg" class="ms-docimage"/>
+
+As soon as the option *Hover* is selected, the user can return to the map where, by hovering the mouse over the objects in the map, the *Floating Identify Tool* shows the relative information. 
```

Comment:
As soon as the option *Hover* is selected, the user can hover the mouse over a layer in the map in order to show the popup containing the identify information.